src/ts/subtitleutils.ts

```diff
@@ -0,0 +1,65 @@
+import {ListSelector, ListSelectorConfig} from './components/listselector';
+import {SubtitleListBox} from './components/subtitlelistbox';
+import {SubtitleSelectBox} from './components/subtitleselectbox';
+
+/**
+ * Helper class to handle all subtitle related events
+ *
+ * This class listens to player events as well as the `ListSelector` event if selection changed
+ */
+export class SubtitleSwitchHandler {
+  private player: bitmovin.PlayerAPI;
```

Comment:
Please always add a blank line between the `class` declaration and the first field/constructor/method/etc.